src/panels/lovelace/hui-editor.ts

```diff
@@ -92,10 +91,10 @@ class LovelaceFullConfigEditor extends LitElement {
           <ha-code-editor
             mode="yaml"
             autofocus
-            .rtl=${computeRTL(this.hass)}
```

Comment:
why remove this computation of dir here??

Reply:
Because code editors should always be LTR and now it's being forced on all of them.

Otherwise it looks like this which is impossible to edit:

![image](https://user-images.githubusercontent.com/37745463/150908759-2b186856-e240-4b0e-ab2e-176f640969f4.png)